Docker/README.md

```diff
@@ -1,19 +1,25 @@
-## Notes:
-1. You can use the polar.usc.edu elastic data, or your local elastic data to begin with the process. Replace the polar URL with your local elastic index URL to do so.
-2. Add your files (pdfs) to the `data/polar/files` folder to extract insights.
+This is the Dockerized version of Polar-deep-insights project. Two parts of the project, insight-generator which is a python library used to extract information and insight-visualizer which is a javascript application used for data visualization can be installed and run on Docker containers using PDI Docker.  
```

Comment:
This is the Dockerized version of the insights generating and insights visualizing components of the EarthCube  Polar-deep-insights project. The insight-generator is a python library used to extract information (names, dates, locations, measurements, etc.) from a collection of files.  The insight-visualizer is a javascript application used for data visualization.  Both components can be installed and run in Docker containers using PDI Docker.